datasets/download_datasets.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from torchtext.datasets import DATASETS, URLS
+
+DATASET_NAMES = ["AG_NEWS", "DBpedia", "IMDB", "SogouNews", "YahooAnswers", "YelpReviewPolarity"]
```

Comment:
this script downloads the above datasets. You can't actually download some of them (bug in torchtext, so you have to do it manually)